src/initialization/MOM_state_initialization.F90

```diff
@@ -1795,7 +1795,7 @@ subroutine initialize_sponges_file(G, GV, US, use_temperature, tv, param_file, C
   if (new_sponges .and. .not. use_ALE) &
     call MOM_error(FATAL, " initialize_sponges: Newer sponges are currently unavailable in layered mode ")
 
-  call MOM_read_data(filename, "Idamp", Idamp(:,:), G%Domain, scale=US%T_to_s)
+  call MOM_read_data(filename, Idamp_var, Idamp(:,:), G%Domain, scale=US%T_to_s)
```

Comment:
On the one hand, I agree that this line should be using Idamp_var, but because the default value for SPONGE_IDAMP_VAR is "IDAMP", not "Idamp", and variable names are case-sensitive, this change will break some experiments.  This change is a good idea, but only if the default for SPONGE_IDAMP_VAR is simultaneously changed to "Idamp".  I am going to try to unilaterally make that change to the default for SPONGE_IDAMP_VAR.

Reply:
I have added the required corresponding change to the default for SPONGE_IDAMP_VAR.